src/browser/BrowserService.cpp

```diff
@@ -611,7 +611,7 @@ QString BrowserService::storeKey(const QString& key)
 
     hideWindow();
     db->metadata()->customData()->set(CustomData::BrowserKeyPrefix + id, key);
-    db->metadata()->customData()->set(QString("%1_%2").arg(CustomData::Created, id),
+    db->metadata()->customData()->set(QString("%1%2").arg(CustomData::Created, id),
```

Comment:
This is re-implementing `getKeyWithPrefix`, any way to use that function or make that more "central"? I almost think that should be a static function of  `CustomData`.

Reply:
I was thinking about that too. Let me check if we use similar kind of prefix inserting elsewhere too.